data/114/190/933/3/1141909333.geojson

```diff
@@ -2,572 +2,576 @@
   "id": 1141909333,
   "type": "Feature",
   "properties": {
-    "edtf:cessation":"uuuu",
-    "edtf:inception":"uuuu",
-    "geom:area":0.0,
-    "geom:area_square_m":0.0,
-    "geom:bbox":"-116.22749,43.60859,-116.22749,43.60859",
-    "geom:latitude":43.60859,
-    "geom:longitude":-116.22749,
-    "iso:country":"US",
-    "lbl:bbox":"-116.24748987,43.5885901073,-116.20748987,43.6285901073",
-    "lbl:latitude":43.60859,
-    "lbl:longitude":-116.22749,
-    "lbl:max_zoom":18.0,
-    "lbl:min_zoom":13.5,
-    "mz:hierarchy_label":1,
-    "mz:is_current":1,
-    "mz:min_zoom":12.0,
-    "name:afr_x_preferred":[
-        "Boise"
+    "edtf:cessation": "",
+    "edtf:inception": "",
+    "edtf:superseded": "2024-05-30",
+    "geom:area": 0,
+    "geom:area_square_m": 0,
+    "geom:bbox": "-116.227490,43.608590,-116.227490,43.608590",
+    "geom:latitude": 43.60859,
+    "geom:longitude": -116.22749,
+    "iso:country": "US",
+    "lbl:bbox": "-116.24748987,43.5885901073,-116.20748987,43.6285901073",
+    "lbl:latitude": 43.60859,
+    "lbl:longitude": -116.22749,
+    "lbl:max_zoom": 18,
+    "lbl:min_zoom": 13.5,
+    "mz:hierarchy_label": 1,
+    "mz:is_current": 0,
```

Comment:
The Natural Earth record was a point without any descendents, so marking it as not current here.